File: codes/Multiprocessing.cpp
#include <thread>
#include <iostream>
#include <chrono>
using namespace std;

# define TASKS 10000

void func(){
this_thread::sleep_for(chrono::milliseconds(1000));
}
int main (){
thread t[TASKS];
for (int i=0;i<TASKS;i++){
t[i]=thread(func);
}
for (int i=0;i<TASKS;i++){
t[i].join();
}
}

File: codes/Multiprocessing.java
import java.util.ArrayList;
import java.util.List;

class Multiprocessing{
    public static void main(String[] args) throws Exception{
        int tasks=10000;
        List<Thread> t=new ArrayList<>();
        for (int i =0;i<tasks;i++){
            Thread thread=new Thread(()->{
                try{
                    Thread.sleep(100);

                }catch(Exception e){
                    System.out.println(e);
                }
            });
            t.add(thread);
            
        }
        for (Thread thread: t){
            thread.join();
        }
            
    }
}
